editor/src/core/shared/dom-utils.ts

```diff
@@ -202,7 +202,8 @@ export const intrinsicHTMLElementNamesThatSupportChildren: Array<string> = [
   'section',
 ]
 
-export function getDOMAttribute(element: HTMLElement, attributeName: string): string | null {
+// eslint-disable-next-line no-restricted-globals
+export function getDOMAttribute(element: Element, attributeName: string): string | null {
```

Comment:
Why the change from `HTMLElement` to `Element`?

Reply:
yeah that's a good question. some code used HTMLElement, some used Element. I think Element covers everything we need, so that's why I changed to the broader type. I can be talked into swapping everything to HTMLElement though

Reply:
I'm fine with either, just wanted to make sure this was a deliberate change